cylc/flow/scheduler.py

```diff
@@ -909,18 +914,45 @@ def process_queued_task_messages(self) -> None:
                     should_poll = True
             if should_poll:
                 to_poll_tasks.append(itask)
-        self.task_job_mgr.poll_task_jobs(
-            self.workflow, to_poll_tasks)
+        if to_poll_tasks:
+            self.task_job_mgr.poll_task_jobs(self.workflow, to_poll_tasks)
+
+        # Remaining unprocessed messages have no corresponding task proxy.
+        # For example, if I manually set a running task to succeeded, the
+        # proxy can be removed, but the orphaned job still sends messages.
+        for _id, tms in messages.items():
+            warn = "Undeliverable task messages received and ignored:"
+            for _, msg in tms:
+                warn += f'\n  {msg.job_id}: {msg.severity} - "{msg.message}"'
+            LOG.warning(warn)
 
     def get_command_method(self, command_name: str) -> Callable:
         """Return a command processing method or raise AttributeError."""
         return getattr(self, f'command_{command_name}')
 
-    def queue_command(self, command: str, kwargs: dict) -> None:
-        self.command_queue.put((
-            command,
-            tuple(kwargs.values()), {}
-        ))
+    def queue_command(
+        self,
+        name: str,
+        args: list,
+        kwargs: dict,
+        log_lines: List[str],
```

Comment:
The message is logged when the command is queued in the scheduler. It has to be formulated in the resolver module because:
- it may include the authorized user name
- several commands have direct methods in the resolver module (i.e. they don't get queued) and they need the same log message

I'll add to the doc-string to explain this.